data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1022,6 +1022,7 @@ amenity-bicycle_rental                              # icon z17- (also has captio
 amenity-bicycle_repair_station                      # icon z17- (also has caption(optional) z17-)
 amenity-bureau_de_change                            # icon z17- (also has caption(optional) z18-)
 amenity-car_wash                                    # icon z17- (also has caption(optional) z17-)
+amenity-compressed_air                              # icon z18- (also has caption(optional) z19-)
```

Comment:
better to put it below `amenity-bicycle_repair_station` or any other "main" tags where a `compressed_air=yes` might appear;
otherwise a compressed air icon might be shown instead of a "main" feature icon

Reply:
Add test that compressed_air has lowest priority among bicycle/car_XXX ?

Reply:
@dvdmrtnz please make its prio lower

Reply:
> @dvdmrtnz please make its prio lower

@pastk Done

> Add test that compressed_air has lowest priority among bicycle_repair/car_repair ?

@vng Where are these tests defined?